internal/jsonschema/testdata/create-constraint-1-invalid-check.txtar

```diff
@@ -0,0 +1,28 @@
+This is an invalid 'create_constraint' migration.
+Check constraint must have the option 'check' configured.
```

Comment:
Do we need further schema tests to check that `no_inherit` is valid only for `CHECK` constraints?